prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -574,20 +599,34 @@ class ActionHistory {
         organization: Organization,
         sendingAt: OffsetDateTime?,
         countToPrint: Int,
-        reportOptions: ReportFunction.Options
+        reportOptions: ReportFunction.Options,
+        reportId: ReportId? = null
     ) {
         jsonGen.writeStartObject()
         // jsonGen.writeStringField("id", reportFile.reportId.toString())   // TMI?
         jsonGen.writeStringField("organization", organization.description)
         jsonGen.writeStringField("organization_id", orgReceiver.organizationName)
         jsonGen.writeStringField("service", orgReceiver.name)
 
+        reportId?.let {
```

Comment:
What is the use case in which a reportId is null?
Would that be if we filtered out EVERYTHING?  in that case, wouldn't we want to write this element?